hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -1128,3 +1129,44 @@ def hpat_pandas_series_le_impl(self, other):
         return hpat_pandas_series_le_impl
 
     raise TypingError('{} The object must be a pandas.series and argument must be a number. Given: {} and other: {}'.format(_func_name, self, other))
+
+
+@overload_method(SeriesType, 'unique')
+def hpat_pandas_series_unique(self, sorted=False):
+    """
+    Pandas Series method :meth:`pandas.Series.unique` implementation.
+
+    .. only:: developer
+
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_unique
+
+    Parameters
+    -----------
+    self: :class:`pandas.Series`
+        input arg
+    sorted: :obj:`bool`
+        return array is sorted
+
+    Returns
+    -------
+    :obj:`numpy.array`
+         returns :obj:`numpy.array` ndarray
+    """
+
+    _func_name = 'Method unique().'
+
+    if not isinstance(self, SeriesType):
+        raise TypingError(
+            '{} The object must be a pandas.series. Given self: {}'.format(_func_name, self))
+
+    def hpat_pandas_series_unique_impl_sorted(self):
```

Comment:
Please, do not forget to use correct parameters for `_impl` function. Example https://github.com/numba/numba/blob/master/numba/unicode.py#L741
Also, check for parameter is boolean